### PR TITLE
[bugfix][FileBase]update optimize hive add kerberos auth

### PR DESCRIPTION
**Pull request by chl-wxp**
## Check list

* [ ] fix hive connector kerberos bug

### CODE REVIEW

File: seatunnel-connectors-v2/connector-hive/src/main/java/org/apache/seatunnel/connectors/seatunnel/hive/utils/HiveMetaStoreProxy.java [hunk 1]
```diff
@@ -51,11 +55,42 @@ private HiveMetaStoreProxy(Config config) {
                 && config.hasPath(BaseSourceConfig.KERBEROS_KEYTAB_PATH.key())) {
             String principal = config.getString(BaseSourceConfig.KERBEROS_PRINCIPAL.key());
             String keytabPath = config.getString(BaseSourceConfig.KERBEROS_KEYTAB_PATH.key());
+            String hdfsSitePath = config.getString(BaseSourceConfig.HDFS_SITE_PATH.key());
+            String kerberosKrb5ConfPath =
+                    config.getString(BaseSourceConfig.KERBEROS_KRB5_CONF_PATH.key());
+            System.setProperty("java.security.krb5.conf", kerberosKrb5ConfPath);
+            // System.setProperty("krb.principal", "hadoop");
```
Comment (ic4y) [inline seatunnel-connectors-v2/connector-hive/src/main/java/org/apache/seatunnel/connectors/seatunnel/hive/utils/HiveMetaStoreProxy.java]: delete it

**Review by TyrantLucifer — CHANGES_REQUESTED** [review]
Thank you for your contribution. I have a question, did you test it？If yes, please offer test screenshots.

**Comment by chl-wxp** [comment]
file 123.conf：
image
image

run：
image
result：
image
@TyrantLucifer

**Review by hailin0 — COMMENTED** [review]
LGTM

File: seatunnel-connectors-v2/connector-hive/src/main/java/org/apache/seatunnel/connectors/seatunnel/hive/utils/HiveMetaStoreProxy.java [hunk 2]
```diff
@@ -53,7 +55,22 @@ private HiveMetaStoreProxy(Config config) {
                 && config.hasPath(BaseSourceConfig.KERBEROS_KEYTAB_PATH.key())) {
             String principal = config.getString(BaseSourceConfig.KERBEROS_PRINCIPAL.key());
             String keytabPath = config.getString(BaseSourceConfig.KERBEROS_KEYTAB_PATH.key());
+            String hdfsSitePath = config.getString(BaseSourceConfig.HDFS_SITE_PATH.key());
+            String kerberosKrb5ConfPath =
+                    config.getString(BaseSourceConfig.KERBEROS_KRB5_CONF_PATH.key());
+            System.setProperty("java.security.krb5.conf", kerberosKrb5ConfPath);
```
Comment (hailin0) [inline seatunnel-connectors-v2/connector-hive/src/main/java/org/apache/seatunnel/connectors/seatunnel/hive/utils/HiveMetaStoreProxy.java:61]: When to use this variable

1. This variable will take effect globally on the current server, and multiple tasks will overwrite this path with each other.
2. This variable is only initialized on the seatunnl-client or task-committer

**Pull request closed without merging**